lib/components/textEditor/buttonStyle.css

```diff
@@ -1,6 +1,6 @@
 .button {
   background: transparent;
-  padding: var(--vds-spacer-xxs);
+  padding: 1px;
   border: 1px solid var(--vds-color-slate-lighter);
```

Comment:
The mock has no borders around the RichText buttons, but it looks like in your images here we have borders.

Reply:
But in mockup we have a comment - "These are the only 6 icons that should be turned on. They should retain the box around each icon like currently in prod though." 
https://virtru.invisionapp.com/console/SS-Secure-Reply-cl3amq7bn000f018c73z15f0p/cl3dbghbl0hhx01yx03mx3mbr/play

it seems that we do not remove the border of the buttons

Reply:
Gotcha, sounds good. :+1: